components/combobox/__tests__/__snapshots__/combobox.snapshot-test.jsx.snap

```diff
@@ -360,6 +362,113 @@ exports[`Base Custom Menu Item Open HTML Snapshot 1`] = `
 </div>"
 `;
 
+exports[`Base Label Required DOM Snapshot 1`] = `
+<div
+  className="slds-form-element"
+>
+  <label
+    className="slds-form-element__label"
+    htmlFor="combobox-unique-id"
+  >
+    <abbr
+      className="slds-required"
+      title="required"
+    >
+      *
+    </abbr>
+    Search
+  </label>
+  <div
+    className="slds-form-element__control"
+  >
+    <div
+      className="slds-combobox_container"
+    >
+      <div
+        aria-describedby="description-unique-id-error-message"
+        aria-expanded={false}
+        aria-haspopup="listbox"
+        aria-owns="combobox-unique-id-listbox"
+        className="slds-combobox slds-dropdown-trigger slds-dropdown-trigger_click ignore-react-onclickoutside slds-has-error"
+        role="combobox"
+      >
+        <div
+          className="slds-combobox__form-element slds-input-has-icon slds-input-has-icon_right"
+          role="none"
+        >
+          <input
+            aria-activedescendant={null}
+            aria-autocomplete="list"
+            aria-controls="combobox-unique-id-listbox"
+            aria-describedby={undefined}
```

Comment:
` aria-describedby="description-unique-id-error-message"` should be here instead despite all the ARIA states up on that `div`.

![screen shot 2018-03-12 at 5 16 16 pm](https://user-images.githubusercontent.com/1290832/37314357-411e61e8-2619-11e8-8669-5067fdd4fbb7.png)